inst/extdata/installPackages.R

```diff
@@ -34,18 +60,59 @@ installPackages <- function(pkgs, lib, repos = getOption("repos")) {
   }
 }
 
-lib <- Sys.getenv("R_LIBS")
-if (nchar(lib) == 0) {
-  stop("Could not get a value for R_LIBS environment variable")
+nodename <- Sys.info()["nodename"]
```

Comment:
It's in there because it's a pain to develop this patch without it. I don't want an `R_LIBS` environment variable lurking around on my development machine (there was a time, in the not-so-distant past, when creating dependencies on environment variable values was considered poor form).

That said, I'll remove it.